changelogs/unreleased/oss-fuzz-in-ci.md

```diff
@@ -0,0 +1,3 @@
+## feature/testing
+
+* Run fuzzing testing continuously on per-push basis (gh-1809).
```

Comment:
I would include a changelog entry right into a commit that implement a feature / bugfix. This way it improves the following access scenarious:

1. When I looking into the commit, I can read a note that is written for a user and so better understand a nature of the changes.
2. I can look over `git log` of a changelog entry and easily find a commit, where a particual change was introduced.

Reply:
not a problem, commits squashed to a single one

Reply:
Thanks!